backend/modules/persistance/domain/commands.py

```diff
@@ -0,0 +1,9 @@
+from dataclasses import dataclass
+
+from ...data.domain import commands as common_commands
```

Comment:
from ...data.domain import commands as common_commands, value_objects as data_value_objects


or it's `isort` which make it like this?

Reply:
no, it was me. Apparently isort doesn't merge import lines. I'll fix it.

Reply:
okay now it's not a problem anymore, because we moved common commands to common module.